src/ui/server/controllers/insurance/account/password-reset/index.ts

```diff
@@ -85,7 +86,18 @@ export const post = async (req: Request, res: Response) => {
       return res.redirect(LINK_SENT);
     }
 
-    return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    // no account found
+    validationErrors = accountDoesNotExistValidation();
```

Comment:
I don't fully understand what this is for. Some more documentation?

Reply:
OK, will add documentation soon - have some other tech improvements todo, added to the list.

This is to render a validation error on the page if no account is found